### DISCUSSION TITLE
How to change/select the environment in the flow

### DISCUSSION THREAD

**Discussion by salokhan** [postmanlabs/postman-flows#111] [Q&A]
I want to change/select the relevant environment at the start of the flow, Is there any way to do it?

**Comment by samay-v** [comment]
Currently, you can [select an environment](https://learning.postman.com/docs/sending-requests/managing-environments/#selecting-an-active-environment) to be active and flows will resolve your variables according to that.
But soon, you'll be able to select environments for individual requests too!
